Algorithm.Python/Alphas/GreenblattMagicFormulaAlgorithm.py

```diff
@@ -0,0 +1,252 @@
+# QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+# Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from clr import AddReference
+AddReference("System")
+AddReference("QuantConnect.Common")
+AddReference("QuantConnect.Algorithm")
+AddReference("QuantConnect.Indicators")
+AddReference("QuantConnect.Algorithm.Framework")
+
+from System import *
+from QuantConnect import *
+from QuantConnect.Orders import *
+from QuantConnect.Algorithm import QCAlgorithm
+from QuantConnect.Python import PythonQuandl
+
+from QuantConnect.Data.UniverseSelection import *
+from QuantConnect.Indicators import *
+from Selection.FundamentalUniverseSelectionModel import FundamentalUniverseSelectionModel
```

Comment:
Some of these imports are not being used in this algorithm:
- `from QuantConnect.Orders import *`
- `from QuantConnect.Algorithm import QCAlgorithm`
- `from QuantConnect.Python import PythonQuandl`
and add
- `from QuantConnect.Orders.Fees import ConstantFeeModel`